app/javascript/components/greeting.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { fetchGreeting } from "../redux/reducer/greetingSlice";
+
+function Greeting() {
+  const dispatch = useDispatch();
+  const chooseGreeting = useSelector((state) => state.greeting);
+  console.log(chooseGreeting);
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+
+  return (
+    <>
+      <h1>If you want to see diferent greeting, than refresh the page 😊.</h1>
+      <ul>
+        {chooseGreeting.map((greeting) => (
```

Comment:
This component should render a random greeting on the screen. The app shows a blank page. Kindly make sure the app works as expected